app/javascript/crayons/Button/Button.jsx

```diff
@@ -121,10 +121,13 @@ Button.defaultProps = {
   onBlur: undefined,
   tabIndex: undefined,
   title: undefined,
+  tagName: 'button',
```

Comment:
I understand the warning here, but it'd be nice for it to know default parameter values in the function signature. I wonder if there's an eslint plugin. 🤔

Reply:
Yeah you're right. For what it's worth, I prefer having the default props in the function signature (since it's closer to where I'm adding code 😄 ). I'll take a look and see if there's something we can configure for this 👍